src/jit/morph.cpp

```diff
@@ -1579,6 +1595,8 @@ void fgArgInfo::ArgsComplete()
                 // Spill multireg struct arguments that have Assignments or Calls embedded in them
                 curArgTabEntry->needTmp = true;
             }
+#ifndef _TARGET_ARM_
+            // TODO: This implementation is not tested on ARM32
             else
```

Comment:
As I commented (https://github.com/dotnet/coreclr/pull/12050#discussion_r120794799), I think it's better to remain it as TODO comment.